int-tests/007-min-alive-zero-update.t

```diff
@@ -0,0 +1,32 @@
+This test checks the behaviour of an update with max-growth zero
+
+Setup unit file
+
+  $ GROUP=007-min-alive-zero-update
+  $ mkdir $GROUP
+  $ echo "[Unit]\nDescription=Unit 1\n[Service]\nExecStart=/bin/bash -c 'while true; do echo Hello %n; sleep 10; done'\n" > $GROUP/$GROUP-unit@.service
+
+Submit and start group
+
+  $ inagoctl --fleet-endpoint=${FLEET_ENDPOINT} submit $GROUP 2 > /dev/null 2>&1
+  $ inagoctl --fleet-endpoint=${FLEET_ENDPOINT} start $GROUP > /dev/null 2>&1
+  $ sleep 3
+
+Modify unit
+
+  $ echo "[Unit]\nDescription=Unit 1 (CHANGED)\n[Service]\nExecStart=/bin/bash -c 'while true; do echo Hello %n; sleep 10; done'\n" > $GROUP/$GROUP-unit@.service
+
+Update unit
+
+  $ inagoctl --fleet-endpoint=${FLEET_ENDPOINT} update $GROUP --max-growth=0 --min-alive=0 --ready-secs=5
```

Comment:
There is currently no other tests that sets the `ready-secs` flag to something other than the default (30).
Setting it to 5 is arbitrary value less than 30. No particular reason, more a gut feeling.